master/buildbot/reporters/gerrit.py

```diff
@@ -300,6 +300,9 @@ def buildComplete(self, key, build):
             result = _handleLegacyResult(result)
             self.sendCodeReviews(build, result)
 
+    def checkConfig(self, *args, **kwargs):
```

Comment:
Thanks for the root causing. I think we should rather revert this hunk.

I did not realize that this would put name as the first positional argument. :-(

what if we use kwargs.pop('name') instead?